workshop/content/40-dotnet/50-table-viewer/400-expose-table.md

```diff
@@ -23,8 +23,8 @@ namespace CdkWorkshop
 
     public class HitCounter : Construct
     {
-        public IFunction Handler { get; };
-        public Table MyTable { get; };
+        public readonly IFunction Handler { get; };
+        public readonly Table MyTable { get; };
```

Comment:
Should either be `readonly` or auto getter with `{ get }` - having both causes this compile error:

    The modifier 'readonly' is not valid for this item [CdkWorkshop]csharp(CS0106)

If `{ get }` then remove trailing semicolon or this compile error:

    Semicolon after method or accessor block is not valid [CdkWorkshop]csharp(CS1597)

It should probably just match what is in [HitCounter.cs L16-L17](https://github.com/aws-samples/aws-cdk-intro-workshop/blob/master/code/csharp/pipelines-workshop/src/CdkWorkshop/HitCounter.cs#L16-L17)

```csharp
        public readonly Function Handler;
        public readonly Table MyTable;
```

Reply:
Nice catch. Done 👍🏻